Hazel/src/Hazel/Core/EntryPoint.h

```diff
@@ -13,7 +14,7 @@ int main(int argc, char** argv)
 
 	auto app = Hazel::CreateApplication();
 	app->Run();
-	delete app;
+    delete app;
```

Comment:
Convert spaces to tabs
```suggestion
	delete app;
```